configure.ac

```diff
@@ -5,6 +5,8 @@ LT_INIT
 AC_CONFIG_SRCDIR([libass/ass.c])
 AC_CONFIG_HEADERS([config.h])
 
+AC_CANONICAL_HOST
+
```

Comment:
nit: placing this immediately before or after `LT_INIT` imho makes more sense. Though possibly moving `LT_INIT` itself after `AC_CONFIG_HEADERS` might also be a good idea to avoid switching between autoconf setup and libtool

unrelated: while it still shows p as “verified”,  GitHub informs me the key you used to sign the commit has expired

Reply:
I believe I changed it as you suggested and thanks for pointing out my GPG key was expired, I updated it locally a while ago and forgot to update it for github too.